log/log.go

```diff
@@ -55,6 +58,7 @@ func CleanExit(format string, v ...interface{}) {
 func Err(format string, v ...interface{}) {
 	fmt.Fprint(Stderr, pretty.Colorize("{{.Red}}[ERROR]{{.Default}} "))
 	fmt.Fprintf(Stderr, appendNewLine(format), v...)
+	ErrorState = true
```

Comment:
for posterity can you add a note in the godoc that this func is not concurrency safe?